proto/tendermint/propagation/types.proto

```diff
@@ -38,10 +39,15 @@ message RecoveryPart {
   bytes  data   = 4;
 }
 
+message Proposal {
+  tendermint.types.Proposal proposal = 1;
+}
```

Comment:
this is stupid, but similar to what the consensus reactor does in order to wrap types